crates/core/src/network/reducers/init.rs

```diff
@@ -41,13 +41,18 @@ pub fn reduce_init(state: &mut NetworkState, root_state: &State, action_wrapper:
     if let Some(sim2h_url) = maybe_sim2h_url_override {
         // ..and we're configured to use sim2h...
         if let BackendConfig::Sim2h(sim2h_config) = &mut p2p_config.backend_config {
-            info!(
-                "Found property 'sim2h_url' in DNA {} - overriding conductor wide sim2h URL with: {}",
-                dna.address(),
-                sim2h_url,
-            );
-            // ..override the conductor wide setting.
-            sim2h_config.sim2h_url = sim2h_url;
+            if (std::env::var("HC_IGNORE_SIM2H_URL_PROPERTY").is_err()) {
+                info!(
+                    "Found property 'sim2h_url' in DNA {} - overriding conductor wide sim2h URL with: {}",
+                    dna.address(),
+                    sim2h_url,
+                );
+                // ..override the conductor wide setting.
+                sim2h_config.sim2h_url = sim2h_url;
+            } else {
+                debug!(
+                    "DNA has 'sim2h_url' override property set, but it's ignored because HC_IGNORE_SIM2H_URL_PROPERTY is set");
+            }
```

Comment:
Does this mean we don't really care what value HC_IGNORE_SIM2H_URL_PROPERTY is set to, only as long as it exists in our env variables?

Reply:
good point, I was being lazy and sloppy.  Fixed.